config/initializers/audit.rb

```diff
@@ -5,3 +5,10 @@
 
 Audit = Audited.audit_class
 Audit.send(:include, AuditExtensions)
+
+# if this is a migration rake task we disable auditing in particular models
+if Foreman.in_rake?('db:migrate')
+  [ Host::Managed, Hostgroup, ProvisioningTemplate, Ptable, User ].each do |klass|
```

Comment:
Or keep it as it is in this PR (this works and looks best to me, autoloader loads the class, audited is called, auditing is set accordingly). I wanted to avoid wrapping `audited` method because if they change the place in the gem where this is being set, we'll miss it. But if there's a consensus on using `alias_method_chain` I'll change it. @eLobato @ShimShtein or anyone else?